src/odemis/acq/move.py

```
@@ -1888,7 +1891,7 @@ def _transform_from_fm_to_sem(self, pos: Dict[str, float]) -> Dict[str, float]:
     def _do_cryo_switch_sample_position(self, future, target):
         try:
             try:
-                target_name = POSITION_NAMES[target]
+                target_name = target.value
             except KeyError:
                 raise ValueError(f"Unknown target '{target}'")
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)